java/pending_code_changes.txt

```diff
@@ -1 +1,2 @@
-
+Code changes:
+ - Make metadata loading thread-safe in MultiFileMetadataSourceImpl.
```

Comment:
this implies it wasn't before -- I think it probably was, but this simplifies everything enormously :)